docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf

```diff
@@ -472,6 +492,8 @@ name ::= '<Text Literal>' ;
 
 column_names ::= column_name { ',' column_name } ;
 
+condition = expression ;
```

Comment:
```suggestion
condition ::= expression ;
```

Reply:
Both versions are correct, trying to migrate to just `=` as it is simpler.